dbux-data/src/dataProviderUtil.js

```diff
@@ -376,7 +376,8 @@ export default {
     }
     else {
       // not a call related trace
-      return null;
+      return trace;
```

Comment:
fixed #122 for getters and setters, and, in general, will not cause the error anymore